index.html

```diff
@@ -198,13 +199,71 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='de'>
+
+				<p>Wir sind eine Gruppe von Einzelpersonen, die ein erhebliches Interesse an der Entwicklung und Gesundheit des World Wide
+					Web ("das Web") haben, und <strong>wir sind zutiefst besorgt</strong> über <a href='https://ampproject.org'>Accelerated Mobile Pages</a> ("AMP"), ein Google-Projekt, das angeblich darauf abzielt, die Benutzererfahrung des Webs
```

Comment:
Benutzererfahrung is a literal translation of user experience, I'd rather use Nutzererlebnis since it's not about experience as in "professional experience" but as in "experiencing something"

Reply:
👍